src/components/SortableCommentList.js

```diff
@@ -177,67 +178,65 @@ export class SortableCommentListComponent extends Component {
     const pluginContent = showCommentList && displayVisualization ? this.renderPluginContent() : null;
     return (
       <div>
-        {canComment || showCommentList ?
-          <div className="sortable-comment-list">
-            {commentForm}
-            <div>
-              {showCommentList &&
-              <h2>
-                <FormattedMessage id="comments" />
-                <div className="commenticon">
-                  <Icon name="comment-o" />&nbsp;{get(sectionComments, 'count') ? sectionComments.count : '0'}
-                </div>
-              </h2>}
-              {pluginContent}
-              {showCommentList &&
-                <div className="row">
-                  <form className="sort-selector">
-                    <FormGroup controlId="sort-select">
-                      <ControlLabel>
-                        <FormattedMessage id="commentOrder" />
-                      </ControlLabel>
-                      <div className="select">
-                        <FormControl
-                          componentClass="select"
-                          onChange={event => {
-                            this.fetchComments(section.id, event.target.value);
-                          }}
-                          value={get(sectionComments, 'ordering')}
-                        >
-                          {keys(ORDERING_CRITERIA).map(key =>
-                            <FormattedMessage id={key} key={key}>
-                              {(message) => <option value={ORDERING_CRITERIA[key]}>{message}</option>}
-                            </FormattedMessage>
-                          )}
-                        </FormControl>
-                      </div>
-                    </FormGroup>
-                  </form>
-                </div>}
+        {(section.commenting !== 'none' && published) &&
+        <div className="sortable-comment-list">
+          {commentForm}
+          <div>
+            <h2>
+              <FormattedMessage id="comments" />
+              <div className="commenticon">
+                <Icon name="comment-o" />&nbsp;{get(sectionComments, 'count') ? sectionComments.count : '0'}
+              </div>
+            </h2>
+            {pluginContent}
+            {this.state.showLoader && (
+              <div className="sortable-comment-list__loader">
+                <LoadSpinner />
+              </div>
+            )}
+            {showCommentList && !this.state.showLoader &&
```

Comment:
I am thinking here we actually shouldn't require `!this.state.showLoader`.

This is because we would like the comment list header *and* the order filter selection to be visible during loading of comments, so the user can be seen what is being fetched and in what order.